src/hooks/useWallet.ts

```
@@ -0,0 +1,213 @@
+
+        const newAddresses = wagmiAddresses.filter(
+            (addr) => !wallets.some((wallet) => areEvmAddressesEqual(addr, wallet.address))
+        )
+        if (newAddresses.length) {
+            newAddresses.forEach((address) => {
+                addAccount({
+                    accountIdentifier: address,
+                    accountType: interfaces.WalletProviderType.BYOW,
+                    userId: user.user.userId,
+                    connector:
+                      connector && connector.icon
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)